compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/SwingComposeSceneLayer.desktop.kt

```diff
@@ -72,7 +79,7 @@ internal class SwingComposeSceneLayer(
         it.isOpaque = false
         it.background = Color.Transparent.toAwtColor()
         it.size = Dimension(windowContainer.width, windowContainer.height)
-        it.addMouseListener(this)
+        it.addMouseListener(backgroundMouseListener)
 
```

Comment:
> Test: open focusable WINDOW layer without dismissOnClickOutside, try to click on the main content

What about `WINDOW` layer with `dismissOnClickOutside = true`? The background still reacts on clicks:

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/17cdc0c8-3e88-4b18-b9b1-07b63df86ca7

With `dismissOnClickOutside = false` it works as expected.

Reply:
It reacts only after the dialog closes. This PR does not target to align closing on CLICK instead of PRESS outside

Reply:
Agreed to fix in another PR